webextension/scripts/popup.js

```diff
@@ -387,6 +390,46 @@ function show_wikibooks() {
   })
 }
 
+function showFactCheck() {
+  chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+    const url = tabs[0].url
+    const tabId = tabs[0].id
+    chrome.storage.local.get(['fact_check'], (event) => {
```

Comment:
We are not using this (`fact_check`) in this function.

Reply:
true - fixed.